source/connect/advanced-connection-options/general-connection.txt

```diff
@@ -93,3 +106,14 @@ Procedure
 .. seealso::
 
    To disconnect from your deployment, see :ref:`<disconnect-tab>`. 
+
+Example
+-------
+
+The following screenshot shows the :guilabel:`General` tab configured
+for an Atlas cluster with :manual:`SCRAM-SHA-256
+</core/security-scram/>`. Since the connection uses the ``mongodb+srv``
+connection string scheme, the connection automatically enables TLS/SSL. 
+
+.. figure:: /images/authentication/new-atlas-connection.png
```

Comment:
I noticed the other figures specified a figwidth of 662px. Should we add that here as well?